cluster/sync-operator.sh

```diff
@@ -22,7 +22,10 @@ function deploy_operator() {
 }
 
 function wait_ready_operator() {
-    # We have to re-check desired number, sometimes takes some time to be filled in
+    # Wait a little for resources to be created
+    sleep 5
```

Comment:
Thanks

Maybe instead of sleep we can check when the resources are Ready?

Reply:
That's what we do, but when the resource doesn't exist yet, wait for condition fails because the resource is not found.